people/alexiamcdonald.html

```diff
@@ -0,0 +1,9 @@
+<<!DOCTYPE html>
+<html>
+<head>
+    <title>Alexia's Page</title>
+</head>
+<body>
+<p> yo I'm awesome! </p>
```

Comment:
The spaces between the end and beginning tags aren't needed! Also, could you indent the paragraph tag?